src/calendar/internal.tsx

```diff
@@ -38,6 +40,7 @@ export default function Calendar({
   const normalizedStartOfWeek = normalizeStartOfWeek(startOfWeek, normalizedLocale);
   const gridWrapperRef = useRef<HTMLDivElement>(null);
   const [focusedDate, setFocusedDate] = useState<Date | null>(null);
+  const { ariaLabelledby, ariaDescribedby } = useFormFieldContext(rest);
```

Comment:
Why do we need ariaLabelledby and ariaDescribedby from formfield? 
If we do so, we need to change the api description?

Reply:
Good callout. Since Calendar can/will be used not as a direct FormField child element but as part of a larger composed component, it makes sense _not_ to use the FormField context values automatically. I've removed that in the next revision. Thanks!